bin/polymer.js

```diff
@@ -30,6 +30,9 @@ resolve('polymer-cli', {basedir: process.cwd()}, function(error, path) {
   let args = process.argv.slice(2);
   let cli = new lib.PolymerCli(args);
   cli.run().catch((err) => {
+    if (err && err.constructor && err.constructor.name === 'CommandFailure') {
```

Comment:
If this is a normal exit, I think it shouldn't be an exception. Let's change `run()` to return a `Promise<Result>`.